addon/model-data.js

```diff
@@ -318,8 +377,10 @@ export default class M3ModelData {
       }
     }
 
+    this._notifyProjectionProperties(dirtyKeys);
+
     if (notifyRecord) {
```

Comment:
Make this an `else` clause to:
`if (this._notifyProjectionProperties(dirtyKeys)) {}`